src/protobuf-net-data/protobuf-net-data.csproj

```diff
@@ -30,7 +30,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="protobuf-net" Version="3.0.101" />
+    <PackageReference Include="protobuf-net" Version="3.2.45" />
```

Comment:
One more thing, the upgrade of protobuf-net to version 3.2.45 results in the warning "System.Collections.Immutable 7.0.0 doesn't support net461 and has not been tested with it. Consider upgrading your TargetFramework to net462 or later." (since the target frameworks contains net461 again).

It doesn't look like the upgrade is required for the new functionality, and there are also no security issues reported since version 3.0.101. We can leave the version to 3.0.101 and if a user want to use a higher version of protobuf-net on a higher runtime version that should not be a problem, agree?

I'll try to revive the AppVeyor pipeline in the meantime.